mllib/src/test/java/org/apache/spark/mllib/tree/JavaDecisionTreeSuite.java

```diff
@@ -21,6 +21,8 @@
 import java.util.HashMap;
 import java.util.List;
 
+import org.apache.spark.api.java.function.Function;
+import org.apache.spark.mllib.linalg.Vector;
```

Comment:
I guess we need to reorder imports (See https://cwiki.apache.org/confluence/display/SPARK/Spark+Code+Style+Guide#SparkCodeStyleGuide-Imports)